nautobot/core/templatetags/helpers.py

```diff
@@ -137,7 +137,7 @@ def pre_tag(value):
         >>> pre_tag("hello")
         '<pre>hello</pre>'
     """
-    if value is not None:
+    if value:
```

Comment:
I'm a bit concerned about this change, as it means that falsey values like `{}`, `False`, and `0` will get mapped to HTML_NONE now.

Reply:
At the very least it invalidates the example explicitly given in the docstring at lines 135-136 above.

Reply:
@Sridevi-NTC can you try this instead? https://github.com/nautobot/nautobot/pull/7689/files#r2320397251